src/compile/legend.ts

```diff
@@ -228,24 +209,17 @@ export namespace encode {
     let labels:any = {};
 
     if (channel === COLOR) {
-      if (fieldDef.type === ORDINAL) {
-        labelsSpec = extend({
-          text: {
-            scale: model.scaleName(COLOR_LEGEND),
-            field: 'data'
-          }
-        }, labelsSpec || {});
-      } else if (fieldDef.bin) {
+      if (fieldDef.bin) {
         labelsSpec = extend({
           text: {
             scale: model.scaleName(COLOR_LEGEND_LABEL),
-            field: 'data'
+            field: 'value'
```

Comment:
i guess we need **another** follow up PR to change data=>value for all axis/legend stuff?  (File an issue?)